src/content/about/join-us.md

```diff
@@ -1,8 +1,48 @@
 ---
 layout: sub-navigation
 title: Join Us
-description: Find out about our recruitment and onboarding processes.
+description: Find out about our latest recruitment drive
+date: 2023-09-21
```

Comment:
what is this date?

Reply:
that was me trying to see if eleventy can pick up the date programmatically like in the blog section but didn't work. I'll remove it.